cardano-cli/src/Cardano/CLI/Optparse.hs

```diff
@@ -0,0 +1,18 @@
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Cardano.CLI.OptParse
```

Comment:
Lets not create an entire module for this function. Let's put it in `Cardano.CLI.Helpers`